xcode/Subconscious/Shared/Library/UserDefaultsProperty.swift

```diff
@@ -7,22 +7,36 @@
 import Combine
 import Foundation
 
-/// Creates a type-safe getter and setter for a UserDefaults key.
-/// Properties decorated with `@UserDefaultsProperty` will get and set their
-/// value from UserDefaults.
+/// Property wrapper that creates a type-safe getter/setter for a
+/// UserDefaults key.
 ///
-/// The property's default value will be used if no value is found in the
-/// UserDefaults for that key.
+/// Properties decorated with `@UserDefaultsProperty` will get and set their
+/// value from UserDefaults. The property's default value will be used if
+/// no value is found in the UserDefaults for that key.
 ///
 /// You can subscribe to changes in this default via the projected value, using
 /// `$name`, where name is the name of your property. Projected value is a
 /// publisher that emits any time the wrapped value changes.
 @propertyWrapper struct UserDefaultsProperty<Value: Equatable> {
+    /// Create a type-safe accessor for a UserDefaults key.
+    private struct UserDefaultsKey<Value: Equatable> {
```

Comment:
Added this inner accessor type to offer a typesafe wrapper we could use in both the constructor and the consumer-facing property wrapper accessor.